x-pack/plugins/security_solution/public/management/pages/policy/view/tabs/policy_tabs.tsx

```diff
@@ -457,7 +457,7 @@ export const PolicyTabs = React.memo(() => {
       cancelUnsavedChangesModal,
       history,
       policyId,
-      routeState.backLink,
+      routeState?.backLink,
```

Comment:
This is the only change that fixes the issue. Right? I'm not sure what the `nextTabId` is needed for.

Reply:
I think that's only additional testing: `nextTabId` is in the `artifact_page.ts` fixture which is used to click through the artifact tabs on Policy Details page. so with `nextTabId` an additional click to the neighbour tab is added, to cover this situation

Reply:
@ashokaditya What @gergoabraham said :D